### PR TITLE
 vmss create: update default instance size to Standard_DS1_v2

### LINKED ISSUE #5970: VMSS create: switch default vm size from standard_d1_v2 to standard_ds1_v2
//CC: @gatneil  @mayurid @twitchax 
Related with https://github.com/Azure/azure-cli/issues/2030

### PR DESCRIPTION
**Pull request by yugangw-msft**
Fix #5970. A planned breaking change.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/command_modules/azure-cli-vm/HISTORY.rst [hunk 1]
```diff
@@ -5,6 +5,7 @@ Release History
 
 2.0.36
```
Comment (tjprescott) [inline src/command_modules/azure-cli-vm/HISTORY.rst]: This should bump the minor version, not the patch version, yes?

Comment (yugangw-msft) [reply src/command_modules/azure-cli-vm/HISTORY.rst]: You know we had breaking changes before but we never touched the minor version. I am fine with either way but want to confirm we will pick up this policy moving forward.

Comment (troydai) [reply src/command_modules/azure-cli-vm/HISTORY.rst]: Let's bump the minor version this time. We should keep doing this from now on.

Comment (tjprescott) [reply src/command_modules/azure-cli-vm/HISTORY.rst]: I would bump the minor version, but that's not even proper semantic versioning (though it feels *more right*).  @johanste  do you have a preference?

Comment (tjprescott) [reply src/command_modules/azure-cli-vm/HISTORY.rst]: Agree with @troydai

Comment (yugangw-msft) [reply src/command_modules/azure-cli-vm/HISTORY.rst]: So, i have updated to `2.1.0`

**Pull request merged**